### PR TITLE
Properties file

### PR DESCRIPTION
**Pull request by jburel**
Add Properties file
Properties initially in omero.properties

### CODE REVIEW

File: src/main/resources/omero-blitz.properties [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+##
+## For more information, see
+##
+## https://docs.openmicroscopy.org/latest/omero/sysadmins/grid.html
+#############################################
+
+# registry_timeout is the milliseconds which
+# the registry and other services will wait
+# on remote services to respond.
+omero.grid.registry_timeout=5000
```
Comment (mtbc) [inline src/main/resources/omero-blitz.properties]: Nice to end with EOL.

**Comment by jburel** [comment]
@mtbc the upgrade check problem is now sorted.
Could you check it again tomorrow?

**Comment by mtbc** [comment]
https://github.com/openmicroscopy/openmicroscopy/pull/6004#issuecomment-483550964

**Comment by jburel** [comment]
merging for m6

**Pull request merged**